switch_test.go

```diff
@@ -356,7 +355,7 @@ func accept(l net.Listener, done <-chan struct{}, sw *Switch) {
 		}
 		var err1, err2 error
 		nodeInfo := new(NodeInfo)
-		cmn.Parallel(
+		Parallel(
```

Comment:
Ok yeah, I just deleted because there were go-common had been imported twice from the last commit... and all other go-common was using the `.` imported, I'll change now though